test/lint/README.md

```
@@ -59,3 +59,7 @@ git remote add --fetch secp256k1 https://github.com/bitcoin-core/secp256k1.git
 all-lint.py
 ===========
 Calls other scripts with the `lint-` prefix.
+
+lint_ignore_dirs.py
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)